assets/index.html

```diff
@@ -76,12 +77,50 @@
 
             <!-- Decoded section start -->
             <div class="col ms-2">
-              <div class="mb-5">
-                <span class="float-start h3" for="jwtout">Decoded</span>
-                <div
-                  class="form-check"
-                  style="float: right; position: relative"
+              <div class="mb-2 d-flex align-items-start">
+                <h3>Decoded</h3>
+                <!-- hidden text area is used for copying whole decoded json both header and payload to clipboard-->
+                <textarea
+                  id="jwtoutput"
+                  name="jwtoutput"
+                  style="display: none"
+                  readonly
+                ></textarea>
+                <button
+                  id="copyoutput"
+                  type="button"
+                  class="btn btn-default btn-sm"
+                  title="Copy header and payload to clipboard"
+                  data-bs-toggle="tooltip"
+                  data-bs-placement="bottom"
+                  data-bs-title="Copy header and payload to clipboard"
+                  onclick="copyToClipboard('jwtoutput', this)"
                 >
+                  <!-- bootstrap icon clipboard-->
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
```

Comment:
Can we add this as a file to our project rather than reference a webpage?

Reply:
Here is some documentation regarding svg https://developer.mozilla.org/en-US/docs/Web/SVG. https://icons.getbootstrap.com/#usage Embedded Embed your icons within the HTML of your page (as opposed to an external image file).

Reply:
This works without internet. No issue.